__tests__/app.test.js

```diff
@@ -15,7 +15,32 @@ beforeEach(() => {
 })
 
 describe("GET /api/topics", () => {
-    test("status code: 200", () => {
+    test("responds with a status code: 200", () => {
         return request(app).get("/api/topics").expect(200)
     })
-})
\ No newline at end of file
+    test("respond with an array of topic objects each with the following properties: slug, description", () => {
+        return request(app)
+            .get("/api/topics")
+            .then(({ body }) => {
+                const { topics } = body;
+                expect(Array.isArray(topics)).toBe(true);
+                expect(topics.length).toBeGreaterThan(0)
+                topics.forEach((topic) => {
+                    expect(typeof topic.description).toBe("string")
+                    expect(typeof topic.slug).toBe("string")
+                })
+                expect(topics[0].slug).toBe("mitch")
+                expect(topics[0].description).toBe("The man, the Mitch, the legend")
+            });
+    });
+});
+
+// describe("GET /api/banana", () => {
```

Comment:
Not sure if this is meant to be commented out? I'd make sure this is working before committing or deleting it 🙂